RFS/src/test/java/org/opensearch/migrations/parsing/JacksonParserVsReadObjectBenchmark.java

```diff
@@ -0,0 +1,117 @@
+package org.opensearch.migrations.parsing;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.StreamReadFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import org.junit.jupiter.api.Test;
+
+import com.rfs.http.BulkRequestGenerator;
+import com.rfs.http.BulkRequestGenerator.BulkItemResponseEntry;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.Warmup;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+
+public class JacksonParserVsReadObjectBenchmark {
```

Comment:
Thanks for testing this possible approach!  Since it appears the other approach has a substantial performance benefit and will therefore be part of our mainline code, I'm curious what your thinking is for whether/where to keep this unused code around longer term?  Do you envision a scenario where we'd want to use this class in our mainline code in the future, or are you thinking this will be useful as a reference for how to perform benchmarking?

Reply:
I'd be curious the teams thoughts on this topic, this seems like a good Friday demo.

Keeping the unused solution allows revisiting this experiment, otherwise it would be lost to time as I have often seen.  I don't think there is value aside from having it available as an artifact.  If it becomes a burden that sounds like a good case to revisit.

I think have a reference cases is useful to allow us to quickly iterate.

Reply:
I agree, having an example would be helpful.  What would be even better would be instrumenting our existing code for performance testing and tracking the results commit-by-commit.  That's definitely out of scope of this pro-bono PR, though.  Would you be willing to make a JIRA to track that work?  Once we have some in-use perf testing code we can remove this unused example.

Reply:
Created broad issue - https://opensearch.atlassian.net/browse/MIGRATIONS-1948